datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -26,12 +28,15 @@ type integrationGcpStsResource struct {
 }
 
 type integrationGcpStsModel struct {
-	ID                   types.String `tfsdk:"id"`
-	Automute             types.Bool   `tfsdk:"automute"`
-	ClientEmail          types.String `tfsdk:"client_email"`
-	DelegateAccountEmail types.String `tfsdk:"delegate_account_email"`
-	IsCspmEnabled        types.Bool   `tfsdk:"is_cspm_enabled"`
-	HostFilters          types.Set    `tfsdk:"host_filters"`
+	ID                             types.String `tfsdk:"id"`
+	AccountTags                    types.Set    `tfsdk:"account_tags"`
+	Automute                       types.Bool   `tfsdk:"automute"`
+	ClientEmail                    types.String `tfsdk:"client_email"`
+	DelegateAccountEmail           types.String `tfsdk:"delegate_account_email"`
+	HostFilters                    types.Set    `tfsdk:"host_filters"`
+	IsCspmEnabled                  types.Bool   `tfsdk:"is_cspm_enabled"`
+	IsSecurityCommandCenterEnabled types.Bool   `tfsdk:"is_security_command_center_enabled"`
+	ResourceCollectionEnabled      types.Bool   `tfsdk:"resource_collection_enabled"`
```

Comment:
Non-blocking: comparing this with https://github.com/DataDog/dogweb/blob/c76c0f9f0ddfd7c2daf210da53bb368c97b4a0ca/dogweb/lib/validation/schemas/gcp_account_validation.py#L2

- What is `DelegateAccountEmail` vs `ClientEmail` ? My guess is `DelegateAccountEmail == client_id` and `ClientEmail == client_email`
  - Update: read the descriptions below, but still, not sure how `DelegateAccountEmail` can have any affect on the API call we make since it's not a request parameter?
- We're missing `cloud_run_revision_filters` here, not sure if that's on purpose or not.

Reply:
Oh I'm sorry, thought this PR was for our V1 API but it's V2, so comparing to this instead:
https://github.com/DataDog/dogweb/blob/c76c0f9f0ddfd7c2daf210da53bb368c97b4a0ca/dogweb/serializers/integrations/gcp/gcp_account_serializer.py#L45

But still the comments above are valid.

Reply:
DelegateAccountEmail is the STS delegate. (Basically, the principal that needs the token creator role on the client Email).

In this schema, it's a computed value that this provider returns, mainly to allow setup purely through terraform.

Basically:

- <terraform to create GCP service account (including permissions)>
- <terraform using this module  to configure the datadog gcp sts integration (using output from previous)>
- <terraform to give output of integration the token creator role to the service account in first>

Reply:
We have an upcoming task to add `cloud_run_revision_filters`.